docs/CHANGES.md

```diff
@@ -0,0 +1,228 @@
+# v1.0 CHANGES
+
+In Godwoken v0, Godwoken EVM has using an internal address format, users need to integrate web3-provider to handling the conversion between Godwoken internal address and the Ethereum address. Even in most cases the conversion is handled transparently, this incompatiblity still causes some problem in conner cases. Such as, we cannot hanlde the conversion happend inside metamask or some hardware wallets environments because the web3-provider plugin in unsupported in these environments.
```

Comment:
```suggestion
In Godwoken v0, Godwoken EVM was using an internal address format. Users need to integrate web3-provider to handling the conversion between the internal address format and the Ethereum address format. Although in most cases the conversion is handled transparently, this incompatibility still causes problems in some corner cases. Such as, we cannot handle the conversion in metamask or some hardware wallets because the web3-provider plugin is unsupported in these environments.
```